src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdatesReporter.groovy

```diff
@@ -226,24 +228,28 @@ class DependencyUpdatesReporter {
     new DependenciesGroup<T>(dependencies.size(), dependencies)
   }
 
-  protected def buildDependency(String group, String name, String version, String projectUrl) {
-    new Dependency(group, name, version, projectUrl)
+  protected def buildDependency(Coordinate coordinate, Map<String, String> key) {
+    new Dependency(key['group'], key['name'], coordinate.getVersion(), projectUrls[key],
+      coordinate.getUserReason())
   }
```

Comment:
These builders were just acting as passthroughs for the constructor. Given that, it made sense to me to pass fewer arguments in and let the builders translate those arguments into the constructor parameters.